Makefile

```diff
@@ -6,8 +6,36 @@
 #
 # App name.
 APP=fml
+ifeq ($(shell go env GOOS),windows)
+  APP:=$(APP).exe
+endif
 # Enable Go Modules.
 GO111MODULE=on
+# Go ldflags.
+# Set version to git tag if available, otherwise use commit hash.
+# Strip debug symbols and disable DWARF generation.
+# Build static binaries on Linux.
+GO_LDFLAGS=-s -w -X github.com/G-Research/fasttrackml/pkg/version.Version=$(shell git describe --tags --always --dirty --match='v*' 2> /dev/null | sed 's/^v//')
+ifeq ($(shell go env GOOS),linux)
+  GO_LDFLAGS+=-linkmode external -extldflags -static
+endif
+# Go build tags.
+GO_BUILDTAGS=$(shell jq -r '."go.buildTags"' .vscode/settings.json)
```

Comment:
should we put the tags here instead of the vscode project settings? This seems like a more authoritative location, and `jq` would not then be required

Reply:
I'd love to have a more naturally authoritative place than the vscode settings to store this! However I haven't figured out a way to make vscode read the tags from a different place than it's settings and I would like codespaces/devcontainers to just work so that any contributor can pick up the project instantly.
If you figure out a way, then let's go for it!

Reply:
This had been bugging since very early on and your remark pushed to finally figure this out! I've centralised all build tags in `.go-build-tags` in 8047c6e. Thanks for pushing me @suprjinx, much appreciated!